bpdm-pool/src/main/kotlin/org/eclipse/tractusx/bpdm/pool/service/BusinessPartnerBuildService.kt

```diff
@@ -188,26 +189,44 @@ class BusinessPartnerBuildService(
         val legalEntities = legalEntityRepository.findDistinctByBpnIn(bpnsToFetch)
         businessPartnerFetchService.fetchDependenciesWithLegalAddress(legalEntities)
 
-        changelogService.createChangelogEntries(legalEntities.map {
+
+        val requestsByBpn = validRequests.associateBy { it.bpnl }
+
+        val changedLegalEntities = legalEntities
```

Comment:
Please don't do these changes in the BusinessPartnerBuildService but instead incorporate them into the TaskStepBuildService. This service is similar to the BusinessPartnerBuildService but uses Orchestrator Golden record tickets to create and update business partners.